sky/skylet/providers/gcp/node.py

```diff
@@ -290,6 +291,41 @@ def create_instance(
         """
         return
 
+    def resize_disk(self, base_config: dict, instance_name: str) -> Tuple[dict, str]:
```

Comment:
Could we replace this with an abstract method and move the implementation to `GCPCompute` ? In `GCPCompute`, you can replace `resource_v1` with `self.resouce`, so we don't need to build the resources again. Also, we should add some error handling if `GCPTPU` calls the function. (Probably fail early in `sky/resources.py::_try_validate_image_id`)

Reply:
Thanks for the suggestions, I've created the abstract method and moved the implementation to ```GCPCompute```.  Also, I've added some error handling in the most recent commit. Could you confirm whether it is sufficient for ```GCPTPU```?